### PR TITLE
Add New Map: Cavy Hills by smugler5

### PR DESCRIPTION
**Pull request by moreavy**

### CODE REVIEW

**Comment by moreavy** [comment]
Help Needed: I have changes the map's base in the `map.mts` file, but due the `map.conf` file, the bases haven't changes yet.

File: cavy_hills/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+r = 100
+initial_stuff = default:pick_stone,default:torch 6,default:ladder 20
+team.2 = blue
+rotation = z
+name = Cavy Hills
+author = smugler5
+h = 230
+team.1.color = red
+team.2.color = blue
+team.2.pos = (91,33,-40)
```
Comment (JostP) [inline cavy_hills/map.conf:11]: > Help Needed: I have changes the map's base in the `map.mts` file, but due the `map.conf` file, the bases haven't changes yet.

If you change numbers in this line, blue base should move to different location.

**Comment by moreavy** [comment]
WIP this, I'll am busy in IRL and would make changes only after 11th April.

**Comment by Realyg-Hmar** [comment]
I saw the screenshot the map is: really bad, when people go down they will commit suicide or someone will take the opportunity to kill him... i think this map shouldn't be on ctf !. That just my opinion

**Pull request closed without merging**